ocf_data_sampler/numpy_batch/datetime_features.py

```diff
@@ -0,0 +1,42 @@
+"""Datapipes to trigonometric date and time to NumpyBatch"""
+
+import numpy as np
+import pandas as pd
+from numpy.typing import NDArray
+
+
+def _get_date_time_in_pi(
+    dt: pd.DatetimeIndex,
+) -> tuple[NDArray[np.float64], NDArray[np.float64]]:
+    """
+    Change the datetimes, into time and date scaled in radians
+    """
+
+    day_of_year = dt.dayofyear
+    minute_of_day = dt.minute + dt.hour * 60
+
+    # converting into positions on sin-cos circle
+    time_in_pi = (2 * np.pi) * (minute_of_day / (24 * 3600))
+    date_in_pi = (2 * np.pi) * (day_of_year / (365 * 24 * 3600))
```

Comment:
For the future, I think this should be `date_in_pi = (2 * np.pi) * (day_of_year / 365 )`